lms/static/sass/_pearsonx.scss

```diff
@@ -213,9 +213,16 @@ h2 {
         width: 100%;
         height: auto;
       }
+      // Hide the picture for small screens OC-4050
+      @media screen and (max-width: 760px) {
+        display: none;
+      }
     }
     .contact-form-container {
-      flex-basis: 40%;
+      // Don't restrict form width when on small screens with no contact picture
+      @media screen and (min-width: 760px) {
+        flex-basis: 40%;
```

Comment:
@xitij2000 This setting doesn't seem to change the way the form is displayed.

Image hidden, `flex-basis` set to `40%`:

![setting-on](https://user-images.githubusercontent.com/961441/36791072-15a3ad74-1c97-11e8-995a-a566e0aba0cf.png)

Image hidden, `flex-basis` not set to `40%`:

![setting-off](https://user-images.githubusercontent.com/961441/36791167-51d55a9a-1c97-11e8-9cfb-2c51f1b7fe33.png)

Reply:
This setting has a greater impact when dealing with slightly larger screens, where otherwise the form would be squished to the left. My understanding of ``flex-basis: 40%`` is that it will set the initial size of the form to 40% of the parent container's size. On larger screens (under 760px) this means that the form will take up 40% of the width of the screen and be squished to the left. On smaller sized mobile screens, it will go beyond 40% so it still looks fine.

Reply:
@xitij2000 Thanks for the details. Do you have some screenshots to share that illustrates the difference? I tested this again with some other devices and couldn't really tell a difference.

Also:

> On larger screens (**under** 760px) this means that the form will take up 40% of the width of the screen and be squished to the left.

Did you mean larger screens **over** 760px? That would match the constraints set by the media query (`@media screen and (min-width: 760px)`).

Reply:
@itsjeyd I mean larger screens in the range under 760px i.e. when you go to the higher range within 0 to 760, the 40% width starts being restrictive. I'll see if I can demonstrate this via a screenshot, and if not, I'll just revert this change.

Reply:
I've checked on the live server and the impact is minimal. It's only useful between 450 px and 760px.
Flex basis inside @ media:
![screen shot 2018-02-28 at 20 45 22](https://user-images.githubusercontent.com/118837/36795207-69ce3ea6-1cc8-11e8-9f78-44ac675cab21.png)

Unchanged code:
![screen shot 2018-02-28 at 20 45 14](https://user-images.githubusercontent.com/118837/36795214-6cce83c2-1cc8-11e8-9a8b-64b6a7fc104c.png)